srv/salt/ceph/smoketests/tuned/mgr/init.sls

```diff
@@ -0,0 +1,6 @@
+Check tuned for ses-mgr roles:
+  salt.state:
+    - tgt: "I@roles:mgr and not I@roles:storage and not I@roles:mon"
+    - tgt_type: compound
+    - sls: ceph.tests.tuned.mgr
```

Comment:
The problem with the smoke tests I have added is that they are targeted. The test `ceph.tests.tuned.mgr` is expected to fail on nodes where there's an OSD (even if there's also a mgr there). The `tgt` here takes care of that, but if one was to run `ceph.tests.tuned.mgr` manually and targets all nodes (or all nodes that have a mgr), he'll see failures and think that there's something wrong. Thoughts on this? Will people run the tests manually without going through `ceph.smoketests.tuned.*`? If they do, maybe adding a message or a comment in the sls file is enough?

Reply:
The alternative is to have the tests look for which roles exist on a node, and determine which profile it should have.